src/Roslyn.Diagnostics.Analyzers/Core/ImportingConstructorShouldBeObsolete.cs

```diff
@@ -11,11 +11,12 @@
 namespace Roslyn.Diagnostics.Analyzers
 {
     /// <summary>
-    /// The importing constructor for a MEF-exported type should be marked obsolete.
-    ///
+    /// <para>The importing constructor for a MEF-exported type should be marked obsolete.</para>
+    /// <para>
     /// <code>
```

Comment:
⚠️ The `<code>` element should not be placed within a `<para>` element. If this change was automated, it is likely a bug in the tool that made the change.